src/Symfony/Component/Intl/Resources/emoji/composer.json

```diff
@@ -1,4 +1,7 @@
 {
+    "name": "symfony/intl-emoji",
+    "description": "Import Unicode Common Locale Data Repository and dependencies for the Emoji Transliterator Builder",
+    "license": "Unicode-DFS-2016",
```

Comment:
License is required for `composer validate --strict` enforcement, but it was hard to figure out how to find the [composer-accepted](https://getcomposer.org/doc/04-schema.md#license) [SPDX license](https://spdx.org/licenses/) type that relates to the [UNICODE LICENSE V3](https://github.com/unicode-org/cldr/blob/main/LICENSE) from the `unicode-org/cldr` project.

[This page](https://spdx.org/licenses/Unicode-DFS-2016.html) for the `Unicode-DFS-2016` license eventually confirmed for me that https://www.unicode.org/license.txt (UNICODE LICENSE V3) is listed as another page for the same license.

Reply:
Also, to be clear, I see in the Symfony contribution guide for [proposing a change](https://symfony.com/doc/current/contributing/code/pull_requests.html#make-a-pull-request) that a license not marked as MIT will be automatically rejected.

It is worth noting that I am not adding any new non-MIT dependencies, I am simply flagging the actual license of `unicode-org/cldr`, which we are already using in the codebase.

Reply:
This directory is under MIT - but actually we don't care because this is composer.json is just a helper to run a dev command